index.html

```diff
@@ -9222,6 +9225,274 @@ <h3 id="example-2">
+          <div class="note">
+            This implements the pre-delay and the application of the reduction
+            gain.
+          </div>
+          <p>
+            The following algorithm describes the <code>EnvelopeFollower</code>
+            object to be applied to the input signal to produce the gain
+            reduction value. First, a couple variables that persist accross
+            invocation of this algorithm, tied to this particular
+            <code>EnvelopeFollower</code> instance.
+          </p>
+          <ul>
+            <li>Let <var>detector average</var> be a floating point number,
+            initialized to 1.0.
+            </li>
+            <li>Let <var>compression gain</var> be a floating point number,
+            initialized to 1.0.
+            </li>
+          </ul>
+          <p>
+            Now, the actual algorithm to determine <var>reduction gain</var>, in
+            linear gain value, for each sample of input.
+          </p>
+          <ol>
+            <li>Let <var>input signal</var> be the the render quantum of audio
+            to be processed.
+            </li>
+            <li>Let <var>threshold</var>, <var>knee</var> have the value of the
+            <a>AudioParam</a> of the same name, <a href=
+            "#db-to-linear">converted to linear unit</a> sampled at the time of
+            processing of this block (as <a>k-rate</a> parameters).
+            </li>
+            <li>Let <var>ratio</var> have the value of the <code>ratio</code>
+            <a>AudioParam</a>, sampled at the time of processing of this block
+            (as a <a>k-rate</a> parameter).
+            </li>
+            <li>Let <var>attack</var> and <var>release</var> have the value of
+            the <a>AudioParam</a> of the same name, sampled at the time of
+            processing (those are <a>k-rate</a> parameters), mutiplied by the
+            sample-rate of the <a>BaseAudioContext</a> this
+            <a>DynamicsCompressorNode</a> is <a href=
+            "#associated">associated</a> with.
+            </li>
+            <li>For each samples <var>input</var> of <var>input signal</var>,
+            execute the following steps:
+              <ol>
+                <li>Let <var>envelope rate</var> be the result of <a href=
+                "#envelope-rate">computing the envelope rate</a>
+                </li>
+                <li>If the absolute value of <var>input</var> is less than
+                0.0001, let <var>attenuation</var> be 1.0. Else, let
+                <var>attenuation</var> be the value of applying the <a
+                href="#compression-curve">compression curve</a> to the absolute
+                value of <var>input</var>, divided by the absolute value of
+                <var>input</var>.
+                </li>
+                <li>Let <var>decibels per frame</var> be the result of dividing
+                <var>attenuationDB</var> by <var>releaseRate</var>.</li>
```

Comment:
I think you meant to divide attenuation by the quantity known in the C++ code as `sat_release_frames`, which is the number of frames in a release time interval. Note that `releaseRate` is not computed until the following step (and it's computed from the work done in this step!)